pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -449,3 +479,14 @@ func hasInternalIP(machine *machinev1beta1.Machine, memberInternalIP string) boo
 	}
 	return false
 }
+
+func removeBootstrapMemberCount(members []*etcdserverpb.Member) []*etcdserverpb.Member {
```

Comment:
please add a test for that function, I would also think this could be a better fit for our etcdcli.

Reply:
shall i move it to commons ?

Reply:
moved to common and added test